cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/dns/HeadlessServiceDnsBootstrap.scala

```diff
@@ -135,130 +103,126 @@ final class HeadlessServiceDnsBootstrap(discovery: SimpleServiceDiscovery, setti
   import HeadlessServiceDnsBootstrap._
   import context.dispatcher
 
+  // FIXME move/rename these classes, doesn't have to be DNS
```

Comment:
yes good point Neither is it about Headless now as well, which is nice I think...? Headless is a specific k8s term, but all this is very generic now :)